newsroom/templates/base_layout.html

```diff
@@ -158,6 +167,11 @@
 <script src="//cdn.iframe.ly/embed.js" async></script>
 {% endif %}
 
+{% if config.PLYR %}
+<script src="//cdn.plyr.io/3.6.3/plyr.js"></script>
```

Comment:
I have reservations about using this library. It would be best if we could achieve this in pure HTML 5 players (or a player that is not deprecated), but for now it's okay

Reply:
I'm hoping to get some time to do this, I think it will involve disabling the controls and implimenting buttons for the play ffwd, rew, download etc.